### DISCUSSION TITLE
https://codemagic.io/flutter-libraries-ebook/ url is broken, not able to download the pdf

### DISCUSSION THREAD

**Discussion by anmolthedeveloper** [codemagic-ci-cd/codemagic-docs#2810] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Visit https://codemagic.io/flutter-libraries-ebook/
2. You'll be redirected to the home page

### Expected results

Get the download page for Flutter libraries we love book

### Actual results

Redirected to home page

**Comment by anmolthedeveloper** [comment, marked as answer]
I have found the correct url: https://blog.codemagic.io/flutter-libraries-ebook-by-codemagic/ebook-flutter-libraries-we-love-by-codemagic.pdf
but this https://codemagic.io/flutter-libraries-ebook/ url is mentioned everywhere (including codemagic's own blog) so it would be helpful for others if you fix it.

**Reply by helinanever** [reply]
Hi @anmolthedeveloper , I'm curious where did you come across the `https://codemagic.io/flutter-libraries-ebook/` URL?

**Reply by anmolthedeveloper** [reply]
Hello @helinanever 
In Flutter's official documentation https://docs.flutter.dev/resources/books and https://blog.codemagic.io/flutter-packages-ebook-update/ also https://blog.codemagic.io/flutter-libraries-ebook-by-codemagic/

**Reply by helinanever** [reply]
Thanks! I'm glad you were able to find the PDF nevertheless. However, please note that it's been last updated 4 years ago. :)
